src/components/ImportExport.js

```diff
@@ -153,65 +202,66 @@ class ImportExport extends React.Component {
           <DialogTitle >Import/Export</DialogTitle>
-          <div>
-            <List>
-              <ListItem>
-                <ListItemText primary='Import' />
-                <input onChange={this.changeFile} type="file" accept=".kml" />
-                <Button variant="raised" onClick={this.importFile} > Import </Button>
-              </ListItem>
-              <li>
-                <Divider inset />
-              </li>
-              <ListItem>
-                <ListItemText primary='Export' />
-                <FormGroup row>
-                  <FormControlLabel
-                    control={
-                      <Radio
-                      checked={selectedExport === 'trails'}
-                      onChange={this.handleSelect}
-                      value='trails'
-                      />
-                    }
-                    label="Trail Layer"
-                  />
-                  <FormControlLabel
-                    control={
-                      <Radio
-                      checked={selectedExport === 'hydrants'}
-                      onChange={this.handleSelect}
-                      value='hydrants'
-                      />
-                    }
-                    label="Hydrants Layer"
-                  />
-                </FormGroup>
+
+          <List>
+            <ListItem divider>
+               <ListItemText primary="Import" />
+               <input onChange={this.changeFile} type="file" accept=".kml" />
+               <Button variant="raised" onClick={this.importFile} > Import </Button>
+            </ListItem>
+
+            <ListItem>
+              <ListItemText primary="Export" />
+              <div style={{ paddingRight: 35 }}>
+                <FormControl className={classes.formControl}>
+                  <Select
+                    value={this.state.selectedExport}
+                    onChange={(e)=> { this.setState({ selectedExport: e.target.value })}}
+                  >
+                    <MenuItem value={'trails'}>Trails</MenuItem>
+                    <MenuItem value={'hydrants'}>Hydrants</MenuItem>
+                  </Select>
+                  <FormHelperText> Layer </FormHelperText>
+                </FormControl>
+
+
+                <FormControl className={classes.formControl}>
+                  <Select
+                    value={this.state.exportType}
+                    onChange={(e)=> { this.setState({ exportType: e.target.value })}}
+                  >
+                    <MenuItem value={'KML'}>KML</MenuItem>
+                    <MenuItem value={'GJ'}>GeoJson</MenuItem>
+                  </Select>
+                  <FormHelperText> Format </FormHelperText>
+                  </FormControl>
+                </div>
                 <Button variant="raised"  onClick={this.exportFile} > Export </Button>
-              </ListItem>
-            </List>
-          </div>
+
+            </ListItem>
+
+          </List>
+
+          <Divider />
+
+
```

Comment:
So much extra lines. I think it must be something about how you usually collapse sections in your text editor and probably don't notice.

Reply:
Check